From 85a5dda3e1ec15ca867ff459256e987d3fc98718 Mon Sep 17 00:00:00 2001
From: Joe MacDonald <joe_macdonald@mentor.com>
Date: Thu, 28 Mar 2019 21:59:18 -0400
Subject: [PATCH] fc/hwclock: add hwclock alternatives

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Joe MacDonald <joe_macdonald@mentor.com>
Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 policy/modules/system/clock.fc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/system/clock.fc b/policy/modules/system/clock.fc
index 4ed95240e..b8e6b801c 100644
--- a/policy/modules/system/clock.fc
+++ b/policy/modules/system/clock.fc
@@ -1,4 +1,5 @@
 /etc/adjtime		--	gen_context(system_u:object_r:adjtime_t,s0)
 
 /usr/bin/hwclock	--	gen_context(system_u:object_r:hwclock_exec_t,s0)
+/usr/bin/hwclock\.util-linux	--	gen_context(system_u:object_r:hwclock_exec_t,s0)
 
-- 
2.34.1

