From 89b022f8aa33ad403b76c3fe53caf1a5fbfe4a4b Mon Sep 17 00:00:00 2001
From: Yi Zhao <yi.zhao@windriver.com>
Date: Fri, 15 Nov 2019 10:19:54 +0800
Subject: [PATCH] fc/brctl: apply policy to brctl alternatives

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 policy/modules/admin/brctl.fc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/admin/brctl.fc b/policy/modules/admin/brctl.fc
index c7cdc3358..231b82dc5 100644
--- a/policy/modules/admin/brctl.fc
+++ b/policy/modules/admin/brctl.fc
@@ -1,2 +1,3 @@
 /usr/bin/brctl	--	gen_context(system_u:object_r:brctl_exec_t,s0)
+/usr/bin/brctl\.bridge-utils	--	gen_context(system_u:object_r:brctl_exec_t,s0)
 
-- 
2.34.1

