From c34bda34ad6855b5a75a6573dcc7b6450b1646d1 Mon Sep 17 00:00:00 2001
From: Yi Zhao <yi.zhao@windriver.com>
Date: Fri, 15 Nov 2019 10:21:51 +0800
Subject: [PATCH] fc/corecommands: apply policy to nologin alternatives

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 policy/modules/kernel/corecommands.fc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/policy/modules/kernel/corecommands.fc b/policy/modules/kernel/corecommands.fc
index 9020ced10..851529abd 100644
--- a/policy/modules/kernel/corecommands.fc
+++ b/policy/modules/kernel/corecommands.fc
@@ -167,6 +167,8 @@ ifdef(`distro_gentoo',`
 /usr/bin/mountpoint\.sysvinit		--	gen_context(system_u:object_r:bin_t,s0)
 /usr/bin/mountpoint\.util-linux		--	gen_context(system_u:object_r:bin_t,s0)
 /usr/bin/nologin		--	gen_context(system_u:object_r:shell_exec_t,s0)
+/usr/bin/nologin\.shadow		--	gen_context(system_u:object_r:shell_exec_t,s0)
+/usr/bin/nologin\.util-linux	--	gen_context(system_u:object_r:shell_exec_t,s0)
 /usr/bin/sash			--	gen_context(system_u:object_r:shell_exec_t,s0)
 /usr/bin/sesh			--	gen_context(system_u:object_r:shell_exec_t,s0)
 /usr/bin/scponly		--	gen_context(system_u:object_r:shell_exec_t,s0)
-- 
2.34.1

