From bc7cf91865d4f49a2af1100e1a0b692fead807e8 Mon Sep 17 00:00:00 2001
From: Yi Zhao <yi.zhao@windriver.com>
Date: Fri, 15 Nov 2019 10:45:23 +0800
Subject: [PATCH] fc/ntp: apply policy to ntpd alternatives

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 policy/modules/services/ntp.fc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/services/ntp.fc b/policy/modules/services/ntp.fc
index ccea79c30..f24cd7c48 100644
--- a/policy/modules/services/ntp.fc
+++ b/policy/modules/services/ntp.fc
@@ -17,6 +17,7 @@
 /run/systemd/timesync(/.*)?			gen_context(system_u:object_r:ntpd_pid_t,s0)
 
 /usr/bin/ntpd				--	gen_context(system_u:object_r:ntpd_exec_t,s0)
+/usr/bin/ntpd\.ntp			--	gen_context(system_u:object_r:ntpd_exec_t,s0)
 /usr/bin/ntpdate			--	gen_context(system_u:object_r:ntpdate_exec_t,s0)
 /usr/bin/sntp				--	gen_context(system_u:object_r:ntpdate_exec_t,s0)
 
-- 
2.34.1

